introduction.md

```diff
@@ -5,3 +5,10 @@ Hi there :) I'm Max.
 This repository is part of the codeschool course 'Mastering Github'
 
 My favorite codeschool path is the JavaScript one. :)
+
+## My Deadly Skills
+
+* Git
+* Ruby
+* JavaScript
+* Vim
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?